lib/api/compiler/class-beans-compiler.php

```diff
@@ -785,26 +785,28 @@ private function remove_modified_files( $hash, $filemtime_hash ) {
 	private function strip_whitespace( $content ) {
 		$replace = array(
 			'#/\*.*?\*/#s' => '', // Strip comments.
+			'#\s\s+#'      => ' ', // Strip excess whitespace.
 		);
 
 		$search  = array_keys( $replace );
 		$content = preg_replace( $search, $replace, $content );
 
-		// Strip all new lines, tabs, and whitespace.
-		$content = str_replace( array( "\r\n", "\r", "\n", "\t", ' ' ), '', $content );
+		// Strip all new lines and tabs.
+		$content = str_replace( array( "\r\n", "\r", "\n", "\t" ), '', $content );
```

Comment:
CHANGED: @christophherr Here, I've removed the space stripper.  This was the buggie.